HOOKS.md

```diff
@@ -23,3 +23,16 @@ Husky supports all git hooks (https://git-scm.com/docs/githooks). Simply add the
 | push-to-checkout | pushtocheckout |
 | update | update |
 | sendemail-validate | sendemailvalidate |
+
+## Advice
+
+Advice on how to use specific hooks.
+
+### commit-msg
+
+Unlike the `commit-msg` git hook, Husky does not pass the commit message in to the `commitmsg` script. Scripts that want to examine the commit message must read the commit message itself. This can be done by reading `.git/COMMIT_EDITMSG`. For example, in Javascript, the commit message script could read the commit message with:
+
+    var vs = require('fs');
+
+    const commitMessage = fs.readFileSync('./git/COMMIT_EDITMSG', 'utf8');
```

Comment:
Should be `.git/COMMIT_EDITMSG'`.  as stated in above paragraph. And yes, this is good advice, if it wasn't for this I would still be trying to figure out how I can the file name since the git documentation says that it is passed as an argument.

Reply:
would be better to use `path.resolve` IMHO, I struggled a bit before using `path` ahah!